src/middleware/istanbulCoverage.js

```diff
@@ -58,7 +78,7 @@ const saveCoverageInfo = (name, info, { root, coverageOutput }) => {
  * Inject coverage post stript to the file content
  * @param {string} file File path where post script should be injected
  */
-const injectPostScript = file =>
+const injectPostScript = (file) =>
```

Comment:
?

Reply:
This is how we should write based on eslint.